inflation: 0.3
inner_r: 0.1
extern_r: 0.1
timeWeight: 7000.0
accWeight: 1.0
jerkWeight: 3.0
maxVel: 0.5
maxAcc: 0.5
maxIts: 23
eps: 0.02
loopHz: 50.0
globalDstX: 0.0
globalDstY: -6.0
globalOriX: -1.0
globalOriY: 0.0
gaussianKernelSize: 5.0
gaussianKernelMiu: 0.0
gaussianKernelSigma: 3.0
